examples/btc-to-evm-fungible-transfer/src/blockstream-api.ts

```diff
@@ -0,0 +1,172 @@
+import { TypeOfAddress } from "@buildwithsygma/bitcoin";
+import type { BitcoinTransferParams } from "@buildwithsygma/bitcoin";
+import type { Network, Signer } from "bitcoinjs-lib";
+import { payments, Psbt } from "bitcoinjs-lib";
+
+type Utxo = {
+  txid: string;
+  vout: number;
+  status: {
+    confirmed: boolean;
+    block_height: number;
+    block_hash: string;
+    block_time: number;
+  };
+  value: number;
+};
+
+type FeeEstimates = Record<string, number>;
+
+const BLOCK_CONFIRMATION_INDEX = "5";
```

Comment:
I believe having this defined within the scope of function is better as this in only used in that function, best place would be after line 25 after 
`    const response = await fetch(`${blockstreamUrl}/fee-estimates`);`